src/DataAccess/DoctrineSubscriptionRepository.php

```diff
@@ -46,7 +46,13 @@ public function countSimilar( Subscription $subscription, \DateTime $cutoffDateT
 			->setParameter( 'cutoffDate', $cutoffDateTime, Types::DATETIME_MUTABLE )
 			->getQuery();
 		try {
-			return intval( $query->getSingleScalarResult() );
+			/**
+			 * PhpStan on level 9 does not like mixed types being passed into intval()
+			 * so we explicitly tell it the type is what we expect to get from Doctrine
+			 * @var float|int|string|null $count
```

Comment:
Since this is an error that will potentially be fixed in a future update with Doctrine, we should mark in in a way that will alert us when it's fixed. Therefore, I'd like to put it in a phpstan [baseline file](https://phpstan.org/user-guide/baseline) rather than having a comment here. Otherwise this comment will potentially stay here forever, esp. if no Doctrine issue is linked,

Reply:
Good idea, forgot about those baselines